app/views/contexts/calculated_summary_context.py

```diff
@@ -32,7 +43,7 @@ def build_groups_for_section(self, section):
                 self._schema,
                 location,
                 self._language,
-                return_to="final-summary",
+                return_to=self.get_return_to(section["id"]),
```

Comment:
```suggestion
                return_to=self.get_return_to(section.get("id")),
```

Reply:
I don't follow.

Reply:
I just meant, section needs to be a dict(?) so we should use dict specific method, but just a detail.